autospotting.go

```diff
@@ -58,7 +59,10 @@ func run(ctx context.Context) {
 		conf.TagFilteringMode,
 		conf.SpotProductDescription)
 
-	xray.Configure(xray.Config{LogLevel: "error"})
+	xray.Configure(xray.Config{
+		ContextMissingStrategy: ctxmissing.NewDefaultLogErrorStrategy(),
```

Comment:
Thanks, this looks much better than I had attempted... I'm just sorry I wasted a few hours trying to fix this myself without any results.

I'll try it out tomorrow.

Reply:
Worked like a charm!